lib/solidus_importer/process_import.rb

```diff
@@ -4,13 +4,19 @@
 
 module SolidusImporter
   ##
-  # This class parse the source file and create the rows (scan). Then it asks to
-  # Process Row to process each one.
+  # This class parses the source file and creates the rows (scan)
+  # Then it processes the Rows grouping by their `entity_id`, by default in the background
   class ProcessImport
-    attr_reader :importer
+    # this in injected mostly to make testing this class more easily.
+    DEFAULT_GROUP_PROCESSOR = ->(import, row_ids) {
+      return ::SolidusImporter::ProcessRowGroupJob.perform_now(import.id, row_ids) if Rails.env.test?
+
+      ::SolidusImporter::ProcessRowGroupJob.perform_later(import.id, row_ids)
```

Comment:
I know that the whole testing/overrides/forks/alouette is a big mess but can't we set the `active_job.queue_adapter` to `:test` to avoid this?

Reply:
Unfortunately I don't know how to do that on a Solidus extension 😅. Removing that line altogether breaks 9 tests. I could try doing some research, though 🤔